dali/operators/reader/loader/video/frames_decoder.cc

```diff
@@ -134,6 +135,23 @@ void FramesDecoder::BuildIndex() {
   Reset();
 }
 
+void FramesDecoder::DetectVfr() {
+  if (NumFrames() < 3) {
+    is_vfr_ = false;
+    return;
+  }
+
+  int pts_step = index_[1].pts - index_[0].pts;
+  for (int frame_id = 2; frame_id < NumFrames(); ++frame_id) {
+    if ((index_[frame_id].pts - index_[frame_id - 1].pts) != pts_step) {
```

Comment:
Just to clarify. PTS is an integer and we are not affected by floating point operation precision.

Reply:
Yes